billing/functions/egress-traffic-queue.js

```diff
@@ -63,3 +75,32 @@ export const handler = Sentry.AWSLambda.wrapHandler(
             }
         }
     })
+
+/**
+ * Sends a record usage to Stripe for a given egress event.
+ * It uses the Stripe API v2023-10-16 to create a usage record for the given subscription item and quantity.
+ * The response is checked to ensure the usage record was created successfully.
+ * 
+ * @param {import('stripe').Stripe} stripe
+ * @param {import('../data/egress').EgressEvent} egressEvent
+ * @returns {Promise<import('@ucanto/interface').Result<boolean, Error>>}
+ */
+async function sendRecordUsageToStripe(stripe, egressEvent) {
+    const subscriptionItem = {
+        id: 'sub_123', // FIXME (fforbeck): 
```

Comment:
@travis, we currently don't have this `subscriptionItem` information available. I was wondering if that would be the exact same ID for all the customers, or if we can find it per customer somehow.

Reply:
it's going to be different for each customer - I think this is the interface you're looking for - it should let you find a Stripe ID (`account` in that interface) given one of our customer IDs (`customer` in that interface) - the naming is a little confusing given that we call "customer IDs" "accounts" in other places - sorry about that!